frontend/packages/dev-console/src/components/import/deployImage-submit-utils.ts

```diff
@@ -252,6 +252,9 @@ export const createOrUpdateDeployment = (
           ],
         },
       },
+      ...(originalDeployment?.spec.strategy.type && {
+        strategy: { type: originalDeployment?.spec.strategy.type },
+      }),
```

Comment:
there might be other information in the strategy object. so we may want to spread the whole `strategy` object here.
example:
```
  strategy:
    type: RollingUpdate
    rollingUpdate:
      maxUnavailable: 25%
      maxSurge: 25%
```